### ISSUE TITLE
Can't send another transaction after sending first one

### ISSUE THREAD

**Issue by slowbackspace** [trezor/trezor-suite#2858]
**Describe the bug**
When I want to send another transaction to the same account as before (tried on BTC testnet n1 on all seed), just with different amount sometimes I get an error. Oh, same thing happens while sending to different address.
`Sign transaction error: -26: txn-mempool-conflict”` each time I hit sent button.

**Info:**
 - Suite Version: tested on web, develop and production
 - Browser: chrome 86.0.4240.111

**Steps to reproduce:**
1. Sent testnet transaction to (I sent it to n4VQ5YdHf7hLQ2gWQYYrcxoE5B7nWuDFNF)
2. change amount (probably doesn't matter) and try to send another tx

**Expected result:**
Second tx is successfully sent

**Actual result:**
Error notification `Sign transaction error: -26: txn-mempool-conflict”`

**Screenshots:**
![image](https://user-images.githubusercontent.com/6961901/97996761-748dcb80-1de8-11eb-8131-64aa2d0d2b37.png)

**Comment by prusnak** [comment]
`txn-mempool-conflict` means you are trying to spend the UTXO of a transaction that is already present in the mempool of the node.

In order to be able to do this, the first transactions needs to have the RBF flag ON which it does not currently.

TL;DR this is expected if the first transactions was not RBF

**Comment by szymonlesisz** [comment]
i think its a race condition between composing new tx and account update process:
1. you did use utxo[0] from account, its sent and blocked by mempool
2. suite waits for either tx notification or new block notification to update this account (which remove uxto[0]) BUT:
3. meanwhile you compose another tx using utxo[0] from not updated account  (it's still available there util 2. finish)

**Comment by szymonlesisz** [comment]
possible solution:
store utxo[0] reference as "spent" in suite (do not use it in next sign tx process) until next account update occurs

**Comment by bosomt** [comment]
**QA OK**

Nixos + Firefox Browser 82
Suite web dev master commit 374d027
Device model T fw 2.3.4 commit 50854b921
